### ISSUE TITLE
do you have predict_proba for logistic_regression with L1 regularization

### ISSUE THREAD

**Issue by Sandy4321** [cvxgrp/cvxpy#765]
your example is without predict_proba parameter
https://www.cvxpy.org/examples/machine_learning/logistic_regression.html

you can see that predict_proba (The returned estimates for all classes are ordered by the label of classes.) is very useful from 
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression.predict_proba

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. 

**Describe the solution you'd like**
A clear and concise description of what you want to happen, and how your solution would solve the problem.

**Describe alternatives you've considered**

**Additional context**
Add any other context.

**Comment by rileyjmurray** [comment]
This can be resolved by looking up the Wikipedia page on logistic regression.

Please post these kinds of questions to Stack Exchange, with a “cvxpy” tag.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
